source/01-LE/02.ptx

```diff
@@ -8,6 +8,40 @@
             </li>
         </ul>
     </objectives>
+
+<subsection>
+    <title>Warm Up</title>
+    <activity>
+        <introduction>
+            <p>
+                Consider the following matrices:
+                <me>A=\left[\begin{array}{ccc|c}
+                    1 &amp; 0 &amp; 3 &amp; 3 \\
+                    3 &amp; -2 &amp; 4 &amp; 0 \\
+                    0 &amp; -1 &amp; 1 &amp; -2
+                    \end{array}\right],\ B=\left[\begin{array}{ccc}
+                    2 &amp; 5 &amp; 3 \\
+                    1 &amp; -2 &amp; 4 \\
+                    3 &amp; -1 &amp; 7
+                    \end{array}\right]</me>
+            </p>
+        </introduction>
+        <task>
+            <statement>
+                <p>
+                    Write down a linear system whose augmented matrix is <m>A</m>. Can you write down another?
+                </p>
+            </statement>
+        </task>
+        <task>
+            <statement>
+                <p>
+                    Write down a linear system whose coefficient matrix is <m>B</m>. Can you write down another?
+                </p>
+            </statement>
+        </task>
```

Comment:
I'm not sure I understand the motivation here. At this stage students are interested in solving systems of equations, and I don't think we've established how considering a coefficient matrix helps in that mission.

Reply:
my purpose here was just to develop definition recall. In particular, this helps them distinguish between augmented/coefficient matrix.

Reply:
Right, but that begs the question, why should students care about coefficient matrices yet? Augmented matrices are a tool to solve linear systems. At this point, why is the student motivated to know what a coefficient matrix is?

Reply:
hmmm. So while it's not in the activity book this early, I do have students consider coefficient matrices in their first problem set. The kinds of questions that I ask them typically involve them seeing what, if anything, they can deduce about two linear systems that share a coefficient matrix, not dissimilar to ideas present in Math Writing Exploration 1.4.8. 

I wonder if there's something short we can add to make it more meaningful? 

One thing that's popping up in my head in this discussion is that strictly speaking, we don't define what a matrix is. In LE2, we motivate row operations as coming from augmented matrices, but then start defining RREF and performing row operation on general matrices. 

I'd almost like to revise this to have them 

1. Write down an example of a matrix (say of fixed dimension). 
2. Write down a linear system that corresponds to this matrix, if we are given the additional info that it is is augmented matrix.
3. Write down a linear system for which the matrix corresponds to its coefficient matrix. 

At this point in the course, I don't know that the exercise is more deep than having students unpack the notation and terminology we've built, but I do find value in that. 

That said, if we did revise this as described above, then I think adding an additional cool-down activity to LE3 could provide space for a pay-off. 

Specifically, I think I would have them explain why if some specific augment matrix corresponds to a system with a unique solution, then this property is true for any other system that shares the same coefficient matrix.

Reply:
I think the first place where investigating a coefficient matrix would be motivated is in LE3. If a coefficient matrix has a zero row, it corresponds to an inconsistent system; if it has a non-pivot column, it corresponds to a consistent system with infinitely-many solutions. If it lacks zero rows and non-pivot columns, then every system it corresponds to is consistent with exactly one solution.

We're kind of tricky about hand waving the definition of a matrix because as @siwelwerd might say, a matrix is just a convenient way to express a linear transformation between Eucliean spaces, and every matrix operation/property we define is defined for linear transformations. But I guess there we are defining the "standard matrix", so it's worth adding something like [Definition 1.1.1](https://teambasedinquirylearning.github.io/linear-algebra/2024e/LE1.html#definition-1) for matrices. Perhaps, the definition could define both a set of vectors (where order doesn't matter) and a matrix of column vectors (where order does matter)?

Reply:
instinctually when I've been teaching (and I think most of my grad students) do the same, we've adopted a mentality of "a matrix is a an array of numbers and it can be interpreted in several different ways" and then highlight them throughout the course (as a system of equations, as an ordered set of vectors, as a linear transformation, as a vector in a vector space, etc.)

So, yes, I think inserting a definition that defines a matrix as an mxn array, and then in the same definition introducting the short hand $$[\vec{a}_1,...\vec{a}_n]$$ if we want to record its columns would be useful.

Reply:
I'm making a PR into this branch to implement this.

Reply:
#387